js/app/dashboard/components/UnassignedTasks.js

```diff
@@ -142,7 +142,7 @@ class UnassignedTasks extends React.Component {
           <Droppable droppableId="unassigned">
             {(provided) => (
               <div className="list-group nomargin" ref={ provided.innerRef } { ...provided.droppableProps }>
-                { _.map(this.props.groups, (group, index) => {
+                { !this.props.toursEnabled && _.map(this.props.groups, (group, index) => {
```

Comment:
I suggest using a conditional (?) rather than && to avoid subtle bugs; https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx